lib/action_controller/serialization/context.rb

```diff
@@ -0,0 +1,12 @@
+module ActionController
```

Comment:
I am fine with whatever is getting consensus :) Looks like I should rename to `active_model_serializers/serialization_context`?